### PR TITLE
[Refactor] Make changes to some pins

### PR DESCRIPTION
**Pull request by ctt-t**
## Types of Changes

- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/citrus/erdnuss65/info.json
[the remark this answers is not in the document]
Comment (ctt-t) [reply keyboards/citrus/erdnuss65/info.json:1]: There is no pcb in use of the old version, should I create a new version

[the remark this answers is not in the document]
Comment (ctt-t) [reply keyboards/citrus/erdnuss65/info.json:1]: > > 旧版本没有 pcb，我应该创建一个新版本吗
> 
> 如果没有旧版本的PCB，那么以这种方式修改它是可以的。
> 
> 您仍然需要正确格式化文件。



> > There is no pcb in use of the old version, should I create a new version
> 
> If there is no PCB of the old version then modifying it this way is fine.
> 
> You will still need to properly format the file.

I don't quite understand how to format, may I ask how to format the file correctly

[the remark this answers is not in the document]
Comment (ctt-t) [reply keyboards/citrus/erdnuss65/info.json:1]: > > There is no pcb in use of the old version, should I create a new version
> 
> If there is no PCB of the old version then modifying it this way is fine.
> 
> You will still need to properly format the file.

I don't quite understand how to format, may I ask how to format the file correctly

**Comment by ctt-t** [comment]
When can this be merge pull requests

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**